### PR TITLE
Added Bjarne's post.

### PR DESCRIPTION
**Pull request by BFedder**
Hi all, I have never opened a PR against a PR, hope this was the correct way to do it... Please let me know if I should add my changes in a different way.

### CODE REVIEW

**Comment by richardjgowers** [comment]
@BFedder yep this is correct

File: _posts/2022-05-27-gsoc-students.md [hunk 1]
```diff
@@ -25,18 +25,22 @@ Add a link to your blog here!
 ## Bjarne Feddersen: Adding Energy Readers to MDAnalysis
 
 <img
-src="Bjarne's Picture Here"
+src="https://media-exp1.licdn.com/dms/image/C4D03AQHiLg970LOylQ/profile-displayphoto-shrink_800_800/0/1590149425816?e=1658966400&v=beta&t=455KVO-rbXV4YqtG7EeO4dQju3-Ar3GDMFtlpFP355I"
```
Comment (richardjgowers) [inline _posts/2022-05-27-gsoc-students.md:28]: I think we'll likely want this in the repo so we're not potentially linking to something else in the future.

Comment (BFedder) [reply _posts/2022-05-27-gsoc-students.md]: Good point - changed

**Review by orbeckst — APPROVED** [review]
LGTM --- do you have a Twitter handle that you want to include here, too?

**Comment by BFedder** [comment]
I don't have a twitter account, no

**Review by IAlibay — APPROVED** [review]
Cheers, we probably won't advertise on linkedin, but hopefully this should direct interested readers to your page :)

**Pull request merged**